config/manifests/bases/manila-operator.clusterserviceversion.yaml

```diff
@@ -6,6 +6,11 @@ metadata:
     capabilities: Basic Install
     features.operators.openshift.io/disconnected: "true"
     features.operators.openshift.io/fips-compliant: "true"
+    features.operators.openshift.io/proxy-aware: "false"
+    features.operators.openshift.io/tls-profiles: "false"
```

Comment:
```
Whether the operator implements well-known tunables to modify the TLS cipher suite used by the operator and, if applicable, any of the workloads it manages.
```
@gouthampacha wdyt about this?

Reply:
Ok looks like the general direction is to set everything to `false` and validate or move to true any relevant parameter later in time.

Reply:
Yes, that is what I think we're recommending for now

Reply:
`operators.openshift.io/infrastructure-features: '["disconnected"]'` bellow is deprecated. we could remove it https://docs.openshift.com/container-platform/4.14/operators/operator_sdk/osdk-generating-csvs.html#osdk-csv-manual-annotations-deprecated_osdk-generating-csvs in a follow up

Reply:
ack thank you, and +1 to remove it in a follow up.